src/pudl/analysis/record_linkage/eia_ferc1_train.py

```diff
@@ -516,6 +515,20 @@ def validate_override_fixes(
     {override_dups.record_id_eia_override_1.unique()}"
     )
 
+    # Make sure there are no duplicate FERC1 id overrides
+    logger.debug("Checking for duplicate FERC1 override ids")
+    assert (
+        len(
+            ferc_override_dups := only_overrides[
+                only_overrides["record_id_ferc1"].duplicated(keep=False)
+            ]
+        )
+        == 0
+    ), (
+        f"Found record_id_ferc1 duplicates: \
+    {ferc_override_dups.record_id_ferc1.unique()}"
+    )
```

Comment:
Discovered that we were not checking this, because we were getting an XPASS in the tests.